src/api/app/policies/user_policy.rb

```diff
@@ -6,4 +6,8 @@ def update?
   def show?
     user.can_modify_user?(record)
   end
+
+  def see_rss?
```

Comment:
This policy method could be replaced by `Token::RssPolicy#create?` since a user can only see their RSS feed if they are allowed to create a token for it.

https://github.com/openSUSE/open-build-service/blob/75a5c61d7d2eb870d35bbf721aece09b0b803da4/src/api/app/policies/token/rss_policy.rb#L7-L9

Reply:
I'm not really sure if you ask for getting rid of  `UserPolicy#see_rss?` or refactoring it.
After some discussion with @OleksandrOrlov and @danidoni , we propose a refactoring on the method, which is now called `rss_access?` in order to avoid including policy's code in the view. Something that is already done in some other policies.

Let us know if you'd propose something different.

Reply:
I meant that this policy method is unneeded.

Thinking more about this, we don't even need a policy for this since the user will be able to see the `RSS for Notifications` link if they have a RSS token. In the view, instead of checking `policy(user).rss_access?`, we can check `user.rss_token`. If it's present, we show the `RSS for Notifications` link.

Reply:
IMO, I think the policy is needed, because other users could visit your profile page.